server/api/config.go

```diff
@@ -154,7 +154,7 @@ func (h *confHandler) updateConfig(cfg *config.Config, key string, value interfa
 		}
 		return h.updateReplicationModeConfig(cfg, kp[1:], value)
 	case "pd-server":
-		return h.updatePDServerConfig(cfg, kp[len(kp)-1], value)
+		return updatePDServerConfig(h.svr, cfg, kp[len(kp)-1], value)
```

Comment:
Why change it?

Reply:
Because I want to use func `updatePDServerConfig` in `admin.go` before.

Reply:
reset it